### PR TITLE
Module

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: include/itkMaxPhaseCorrelationOptimizer.h [hunk 1]
```diff
@@ -46,7 +43,7 @@ namespace itk
  *
  */
 template < typename TRegistrationMethod >
-class ITK_EXPORT MaxPhaseCorrelationOptimizer :
```
Comment (dzenanz) [inline include/itkMaxPhaseCorrelationOptimizer.h]: Shouldn't this be `ITK_TEMPLATE_EXPORT`? The other cases too.

Comment (thewtex) [reply include/itkMaxPhaseCorrelationOptimizer.h]: Yes, good catch.

**Comment by dzenanz** [comment]
My in-line comment gets shown as outdated. `ITK_EXPORT` -> `ITK_TEMPLATE_EXPORT`?

**Comment by thewtex** [comment]
@dzenanz done

**Pull request merged**